crates/ruff_workspace/src/configuration.rs

```diff
@@ -583,6 +585,29 @@ impl Configuration {
                 .chain(selection.unfixable.iter())
                 .chain(selection.extend_fixable.iter())
             {
+                #[allow(deprecated)]
+                if matches!(selector, RuleSelector::Nursery) {
+                    let suggestion = if preview.is_disabled() {
+                        "Use the `--preview` flag instead."
+                    } else {
+                        "Use the `PREVIEW` selector instead."
+                    };
+                    warn_user_once!("The `NURSERY` selector has been deprecated. {}", suggestion);
+                }
+
+                if preview.is_disabled() {
+                    if let RuleSelector::Rule { prefix, .. } = selector {
+                        if prefix.rules().any(|rule| rule.is_nursery()) {
+                            deprecated_nursery_selectors.insert(selector);
+                        }
+                    }
+
+                    // Check if the selector is empty because preview mode is disabled
+                    if selector.rules(PreviewMode::Disabled).next().is_none() {
+                        ignored_preview_selectors.insert(selector);
+                    }
```

Comment:
And this is like -- you selected `CPY`, but preview is disabled?

Reply:
Do both warnings show if you select `CPY001`? (Should they?)

Reply:
Do you mean like https://github.com/astral-sh/ruff/pull/7210/files#diff-92738d4e248765a1e48e3c930a03e20ed306f9b0c23dd67ad72912133dbb10f2R289-R306? Or if you select CPY _and_ CPY001?

Reply:
I meant the former